common/lib/xmodule/xmodule/templates/problem/jsinput_response.yaml

```diff
@@ -161,7 +161,7 @@ data: |
       }
       </script>
         <p>What is the range of passengers that can fit on a 747 aircraft? Set the low and high bounds. (416 and 660 persons)</p>
-        <p class="sr" aria-hidden="true">
+        <p class="is-hidden" aria-hidden="true">
```

Comment:
The entire problem :) Before, we were using that cone and cube example that was only usable by persons who would see and differentiate color. This is a simpler example that uses a custom range slider.